lessons/222/elixir-app/lib/app/telemetry.ex

```diff
@@ -0,0 +1,25 @@
+defmodule App.Telemetry do
+  use Supervisor
+
+  import Telemetry.Metrics
+
+  def start_link(arg) do
+    Supervisor.start_link(__MODULE__, arg, name: __MODULE__)
+  end
+
+  @impl true
+  def init(_arg) do
+    children = [{Peep, [name: :app, metrics: metrics()]}]
+    Supervisor.init(children, strategy: :one_for_one)
+  end
+
+  def metrics do
+    [
+      distribution([:bandit, :request, :stop, :duration],
+        tags: [],
+        unit: {:native, :millisecond},
```

Comment:
@yordis  Also, this value needs to be in seconds, not milliseconds. First of all, it's a Prometheus (open telemetry) standard, and second, to visualize metrics on the same graph, they need to use the same time unit."

Reply:
btw, here is the list of telemetry https://hexdocs.pm/bandit/Bandit.Telemetry.html

Reply:
Thanks